node/src/gql/types/lesson.ts

```diff
@@ -37,18 +39,37 @@ export const LessonType = new GraphQLObjectType({
     deleted: { type: GraphQLBoolean },
     id: { type: GraphQLID },
     lessonId: { type: GraphQLString },
-    createdBy: { type: GraphQLString },
     name: { type: GraphQLString },
     intro: { type: GraphQLString },
     question: { type: GraphQLString },
     image: { type: GraphQLString },
     expectations: { type: GraphQLList(LessonExpectationType) },
     conclusion: { type: GraphQLList(GraphQLString) },
     lastTrainedAt: { type: DateType },
+    features: { type: GraphQLJSON },
+    isPrivate: { type: GraphQLBoolean },
+    isTemplate: { type: GraphQLBoolean },
     createdAt: { type: DateType },
     updatedAt: { type: DateType },
     trainingConfig: { type: GraphQLString },
-    features: { type: GraphQLJSON },
+    createdBy: {
+      type: UserType,
+      resolve: async function (lesson) {
+        return await User.findOne({ _id: lesson.createdBy });
+      },
+    },
+    contentManagers: {
```

Comment:
let's avoid having lists of users attached to each lesson if possible. Leaving aside how it should work in the db for a minute, could we change the graphql query to something that takes a userId and returns a set of permissions? e.g.

```graphql
query userPerms(user) {
  view: boolean
  edit: boolean
  ...
}
```

?